src/rebar_plugins.erl

```diff
@@ -62,7 +66,11 @@ install(State, AppInfo) ->
 
     State2 = lists:foldl(fun(Profile, StateAcc) ->
                              Plugins = rebar_app_info:get(AppInfo, {plugins, Profile}, []),
-                             handle_plugins(Profile, Plugins, StateAcc)
+                             %% hack to keep {add, [{plugins, [<plugin>]}]} from attempting
+                             %% to install the added plugin to itself, resulting in a never
+                             %% ending list of plugins to install
```

Comment:
might be interesting to add a warning as well. This would be the only functionality that does this, but it might be good to let people know "this is starting to be dangerous territory" and maybe they can pick which apps they add the plugin to instead.

Reply:
This is being removed, I realized we already are tracking plugin apps. So I can ensure we only do each plugin once instead of doing this.